stable/cert-manager/templates/deployment.yaml

```diff
@@ -14,6 +14,10 @@ spec:
     matchLabels:
       app: {{ template "cert-manager.name" . }}
       release: {{ .Release.Name }}
+  {{- with .Values.strategy }}
+  strategy:
+    {{- . | toYaml | nindent 4 }}
```

Comment:
Think this is meant to say `indent`

Reply:
I thought this too 😄 but it in fact adds a newline along with the indent